dbfit-java/core/src/main/java/dbfit/util/Binary.java

```diff
@@ -0,0 +1,70 @@
+package dbfit.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Binary {
+    private byte[] binaryData;
+
+    public Binary(final byte[] byteStream) {
+        binaryData = byteStream;
+    }
+
+    public int getLength() {
+        return binaryData.length;
+    }
+
+    private byte getByte(int index) {
+        return binaryData[index];
+    }
+
+    public byte[] getBytes() {
+        return binaryData;
+    }
+
+    public static Object parse(String s) throws Exception {
+        List<Byte> bytes = new ArrayList<Byte>();
+        if (!s.substring(0, 2).equals("0x")) {
+            throw new NumberFormatException("Hexadecimal string does not start with 0x");
+        }
+        String input = s.substring(2);
+        for (int i = ((input.length()) / 2); i > 0; i--) {
```

Comment:
I think Apache Commons Codec library already includes hex parsing functionality. Maybe we can use it here?

Reply:
It's good to add some unit tests for this.